.tool-versions

```diff
@@ -1 +1 @@
-nim 1.4.2
+nim 1.2.6
```

Comment:
Nim projects build from source. I want to avoid running into issues in third party libraries related to compiler version incompatibilities and worst, bugs. We will definitely migrate as soon as the core third party libraries such as chronos and libp2p do.